neutron_vpnaas/services/vpn/device_drivers/meter_ipsec.py

```diff
@@ -0,0 +1,336 @@
+# Copyright (c) 2016 Jin Jing Lin, IBM.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+import time
```

Comment:
can we rename this filename? meter_ipsec seems like a kind of ipsec drivers, but actually it is not.
maybe meter_addon.py?